.github/workflows/build.yml

```diff
@@ -56,12 +56,16 @@ jobs:
         portmidi
         protobuf
         pthreads
-        qt5compat
+        qt5compat[qml]
         qtbase
         qtdeclarative
         qtsvg
         qttranslations
         qtkeychain-qt6
+        qt3d[qml]
```

Comment:
Do we only need the QML feature? Than `qt3d[core,qml]` could be used instead, which doesn't build the optional default-features of this VCPKG port.
The build time of qt3d was 39 minutes in the last Windows CI run.

Reply:
I'm not entirely sure we do, I need to test without it, but I suspected this could be related to `QtQuick.Scene3D` used [there](https://github.com/mixxxdj/mixxx/pull/11407/files#diff-f4dc897845c4b6663de2495de147e96b3f7567649a29aecd841f89f45c146d6eR3) but I haven't been able to test on Windows as I seem to get a crash when I try to open a device (could be my VM)